utils/utils_test.go

```diff
@@ -98,6 +98,7 @@ func TestAssertEqual(t *testing.T) {
 		{"error not equal", errors.New("1"), errors.New("2"), false},
 		{"driver.Valuer equal", ModifyAt{Time: now, Valid: true}, ModifyAt{Time: now, Valid: true}, true},
 		{"driver.Valuer not equal", ModifyAt{Time: now, Valid: true}, ModifyAt{Time: now.Add(time.Second), Valid: true}, false},
+		{"driver.Valuer equal (ptr to nil ptr)", (*ModifyAt)(nil), &ModifyAt{}, false},
```

Comment:
In the code of `tx.Statement.Changed(..)` the following statements create the nil pointer I added to the test:

```
type model struct {
	ModifyAt *ModifyAt
}

m := model{}
f := reflect.Indirect(reflect.ValueOf(m)).Field(0)
i := f.Interface()
```
.

I is passed to AssertEqual.